src/main/resources/application.yml

```
@@ -108,6 +112,9 @@ synq:
   path:
     base: ${SYNQ_BASE_URL}
 
+# Secret for the signature on callbacks in stage environment
+callback:
+  secret: ${CALLBACK_SECRET}
```

I prefer to have some room in the sourcecode. 
```suggestion
  secret: ${CALLBACK_SECRET}

```